### DISCUSSION TITLE
blog/2022-12-01-set-your-sigma-prior-when-you-know-very-little-about-your-sum-score-data/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#7] [Comments]
# blog/2022-12-01-set-your-sigma-prior-when-you-know-very-little-about-your-sum-score-data/

In this post, we discuss ways to set a prior for sigma when you know little about your sum-score data. Along the way, we intruduce Popoviciu's inequality, the uniform distribution, and the beta-binomial distribution.

https://solomonkurz.netlify.app/blog/2022-12-01-set-your-sigma-prior-when-you-know-very-little-about-your-sum-score-data/

**Comment by ASKurz** [comment]
Someone recently asked where they could download the PHQ-9 data from this post. They're on [GitHub](https://github.com/ASKurz/blogdown/tree/main/content/blog/2022-12-01-set-your-sigma-prior-when-you-know-very-little-about-your-sum-score-data/data).
